metrics-otlp/metrics-gateway-values.yaml

```diff
@@ -0,0 +1,143 @@
+# possible values are documented here: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml
+
+# Run as gateway
+mode: deployment
+
+# Have a nicer name for the resources
+fullnameOverride: "metrics-gateway"
+
+# actual otel controller configuration
+config:
+  receivers:
```

Comment:
Adding the k8s_cluster receiver might be useful as well.

Reply:
I focused on data relevant for my workload mainly.. I prefer to do more in a later iteration. Or should we then remove the node receiver as well?

Reply:
Then let's keep the node metrics. Does not hurt to have them.